engine/src/main/scala/cromwell/engine/io/IoActor.scala

```diff
@@ -143,7 +164,41 @@ final class IoActor(queueSize: Int,
       val replyTo = sender()
       val commandContext= DefaultCommandContext(command, replyTo)
       sendToStream(commandContext)
-    case BackPressureTimerResetAction => serviceRegistryActor ! LoadMetric("IO", NormalLoad)
+
+    case BackPressure(duration) =>
+      lazy val proposedExpiry = OffsetDateTime.now().plusNanos(duration.toNanos)
+
+      // Because this method will be called every time we backpressure, the timer will be overridden every
+      // time until we're not backpressuring anymore
+      backpressureExpiration match {
+        case None =>
+          // Start a new backpressure
+          val durationSeconds = duration.toMillis / 1000.0
+          log.info("Beginning IoActor backpressure for {} seconds", f"$durationSeconds%.2f")
+          timers.startSingleTimer(BackPressureTimerResetKey, BackPressureTimerResetAction, duration)
+          backpressureExpiration = Option(proposedExpiry)
+
+        case Some(expiry) if expiry.isBefore(proposedExpiry) =>
+          // Extend the current backpressure
+          val extension = expiry.until(proposedExpiry, ChronoUnit.MILLIS)
+          val extensionSeconds = extension / 1000.0
+          // There can be a lot of very short extensions when bursts of I/O requests are rejected from a full I/O
+          // queue. Don't clutter the logs with messages for these.
+          if (extensionSeconds > 1.0) {
+            log.info("Extending IoActor backpressure {} seconds", f"$extensionSeconds%.2f")
+          }
```

Comment:
Is there any chance one would want to change this logging frequency without recompiling the code? If so, then `extensionSeconds` should be configurable too.